system/datatypes/text.php

```diff
@@ -146,9 +146,13 @@ private function editAction(HyphaRequest $request) {
 				return ['redirect', $this->constructFullPath($this->pagename)];
 			}
 
+
 			// create form
 			$form = $this->createEditForm($request->getPostData());
 
+			// Validate
```

Comment:
Comment are written in lower case throughout the application.

Reply:
Heh, never really consciously realized that, but indeed. Fixed now.